chat-server/serverhtml.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Welcome to CYF Chat</title>
+  </head>
+  <body>
+    <h1>CYF Chat</h1>
+    <h2>Send a message</h2>
+    <form id="messageForm" action="/messages" method="post">
+      <p>
+        Name:
+        <input type="text" name="from" id="from" placeholder="Your Name" />
+        <br />
+        Message:
+        <input type="text" name="text" id="text" placeholder="The message..." />
+        <br />
+      </p>
+      <button type="submit">Send</button>
+    </form>
+
+    <a href="/messages">See all messages</a>
```

Comment:
This does display the messages but redirects the user to the endpoint where the messages are so they cant send anymore. Is there possibly a better way of displaying the messages?